data/json/martialarts.json

```diff
@@ -936,6 +936,77 @@
             "tec_venom_snake_bite",
             "tec_venom_snake_strike"
         ]
+    },{
+        "type" : "martial_art",
+        "id" : "style_hema",
+        "name" : "HEMA",
+        "description": "The modern recreation of European fencing, based off the manuals of Medieval and Renaissance fencing masters. It emphasises quick footwork and aggressive striking. Skilled fencers can apply their techniques to make lightning fast attacks and devastating combos, while maintaining their defense.", 
+        "arm_block" : 99,
+        "leg_block" : 99,
+        "onmove_buffs" : [
+            {
+                "id" : "hema_move_buff",
+                "name" : "Quick Footwork",
+                "description" : "You keep your distance from the enemy, giving you +1 bonus dodge and +2 to your dodge skill, as long as you keep moving",
+                "unarmed_allowed" : true,
+                "min_melee" : 2,
+                "buff_duration" : 2,
+                "max_stacks" : 1,
+                "bonus_dodge" : 1,
+                "dodge" : 2
+            }
+        ],
+		"onhit_buffs" : [ 
+		    {
+			    "id" : "hema_hit_buff",
+				"name" : "Aggressive Striking",
+				"description" : "You unleash a flurry of blows on the enemy, increasing your damage and the number of blocks you can make, as long as you keep hitting",
+				"min_melee" : 2,
+				"buff_duration" : 2,
+				"max_stacks" : 1,
+				"cut_mult" : 1.1,
+				"bash_mult" : 1.1,
+				"bonus_blocks" : 1
+			}
+		],
+        "techniques" : [
+			"tec_hema_closein",
+			"tec_hema_strike",
+			"tec_hema_powerstrike",
+			"tec_hema_masterstrike"
+        ],
+        "weapons" : [
+            "broadsword",
+            "diamond_broadsword",
+            "broadfire_off",
+            "broadfire_on",
+            "sword_forged",
+            "rapier",
+            "diamond_rapier",
+            "umbrella",
+            "cane",
+            "cudgel",
+			"q_staff",
+			"i_staff",
+			"spear_rebar",
+			"pointy_stick",
+			"spear_wood",
+			"spear_forked",
+			"spear_steel",
+			"spear_survivor",
+			"pool_cue",
+			"javelin_iron",
+			"javelin",
+			"pitchfork",
+			"sword_wood",
+			"sword_nail",
+			"sword_crude",
+			"diamond_zweihander",
+			"zweihander",
```

Comment:
Yeah I'm thinking this style should be more focused on heavy stuff like the broadsword, zweihander, etc. Making it use the rapier takes away from fencing a bit, and it seems a bit odd for a medieval combat style to make use of a sword that wasn't invented until the 16th century.